docs/config.md

```diff
@@ -332,3 +332,18 @@ export default defineConfig({
     ]
   });
   ```
+
+### sendWarningsToBrowser
+
+- **Type:** `boolean`
+- **Default:** `false`
+
+Sends a websocket message `svelte:warnings` with the warnings that are passed to `onwarn`. This is only useful if you build a custom browser based integration where you want to display these.
+
+**Example**
+
+```js
+import.meta.hot.on('svelte:warnings', (warnings) =>
+  console.log('received svelte warnings', warnings)
+);
```

Comment:
this was just an example. Not suggesting v-p-s injects this itself. users could and they are free to use whatever implementation they need.

Reply:
Yeah I was thinking that v-p-s should inject it by default. For me, the experimental option name kinda implied this behaviour. Not sure how hard it'll be, but maybe it'll end up needing the sveltekit `start.js` hack.

Otherwise we could probably do this later on too since it's additive, in that case maybe we should update the docs as
```suggestion
import.meta.hot.on('svelte:warnings', (payload) =>
  console.log(
    payload.normalizedFilename,
    'received svelte warnings',
    payload.warnings
  )
);
```